src/reducers/movieEdit.js

```diff
@@ -0,0 +1,20 @@
+import {CANCEL_EDIT_MOVIE, START_ADD_MOVIE, START_EDIT_MOVIE} from "../actions";
+
+function getNewMovie() {
```

Comment:
this is redundant. this logic should be inside a MovieForm component which you did not create.